src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -106,6 +106,13 @@ def resize_vmss(name: UUID, capacity: int) -> None:
             capacity,
             err,
         )
+    except HttpResponseError as err:
+        if "limit of 10 models that may be referenced" in str(err):
```

Comment:
let's not have the specific number 10 in here in case it changes for some reason?